src/assets/i18n/de_DE.json

```diff
@@ -798,6 +798,7 @@
   "checkout.order_review.heading.title": "Ihre Bestelldaten prüfen",
   "checkout.order_review.send.button": "Bestellung absenden",
   "checkout.order_summary.heading": "Bestellübersicht",
+  "checkout.payment.addPayment.form.legend": "Parameterformular für Zahlungsmittel {{paymentMethodName}}",
```

Comment:
This is about "Zahlungsmittel" (payment instruments), not "Zahlungsart" (payment method), right? I’m asking because the variable is still called {{paymentMethodName}}.

Also, why is it called "Parameterformular für Zahlungsmittel" in one place and "Formular für Zahlungsmittelparameter" further down? Shouldn’t it be consistent?